e2e_tests/integration/loadcsv.spec.js

```diff
@@ -37,24 +37,22 @@ describe('LOAD CSV', () => {
     }
     cy.executeCommand(':clear')
     cy.executeCommand('MATCH (n) DETACH DELETE n')
-    cy.executeCommand(`LOAD CSV WITH HEADERS FROM 'file:///import.csv' AS row{shift}{enter}
```

Comment:
Cypress uses the {} syntax for special chars and {{} for typing {. Since we send the strings right to redux now and don't type them, they are no longer needed.

Reply:
I'll review properly next week, I just wanted to point out that I don't like the idea if bypassing user interfaces in e2e tests.

I don't know the motivation behind it in this case, but generally speaking e2e tests should mimic user interactions.

Reply:
Then let's wait until you're back! It's from my interpretation of cypress best practices https://docs.cypress.io/guides/references/best-practices.html